pkg/box/box.go

```diff
@@ -34,10 +34,14 @@ type AWSConfig struct {
 	// with AWS.
 	DefaultRole string `yaml:"defaultRole"`
 
-	// DefaultProfile is the default profile to use when communcating
+	// DefaultProfile is the default profile to use when communicating
 	// with AWS.
 	DefaultProfile string `yaml:"defaultProfile"`
 
+	// AccountId is the default Account ID to use when communicating
+	// with AWS.
+	DefaultAccountID string `yaml:"defaultAccountId"`
```

Comment:
suggestion: be consistent with ID case (below it's `federationAppID` and `oidcClientID`)
```suggestion
	DefaultAccountID string `yaml:"defaultAccountID"`
```